queen-attack.json

```diff
@@ -0,0 +1,111 @@
+{
+  "#": [
+    "Testing invalid positions will vary by language. The expected",
+    "value of -1 is there to indicate some sort of failure should",
+    "occur.",
+    "Some languages implement tests beyond this set, such as checking",
+    "for two pieces being placed on the same position, representing",
+    "the board graphically, or using standard chess notation. Those",
+    "tests can be offered as extra credit"
+  ],
+  "cases": [
+    {
+      "description": "can not attack",
+      "white_queen": {
+        "position": "(2,4)"
+      },
+      "black_queen": {
+        "position": "(6,6)"
+      },
+      "expected": false
+    },
+    {
+      "description": "can attack on same rank",
+      "white_queen": {
+        "position": "(2,4)"
+      },
+      "black_queen": {
+        "position": "(2,6)"
+      },
+      "expected": true
+    },
+    {
+      "description": "can attack on same file",
+      "white_queen": {
+        "position": "(4,5)"
+      },
+      "black_queen": {
+        "position": "(2,5)"
+      },
+      "expected": false
+    },
+    {
+      "description": "can attack on first diagonal",
+      "white_queen": {
+        "position": "(2,2)"
+      },
+      "black_queen": {
+        "position": "(0,4)"
+      },
+      "expected": false
```

Comment:
Ditto on the description for this one. I need to learn to defer merging until I have my full wits to think through things more carefully!